hypermap/search_api/views.py

```diff
@@ -6,6 +7,12 @@
 from .serializers import SearchSerializer
 import json
 
+import sys  
+
+#Set utf8 as coder and encoder string 
+reload(sys)  
+sys.setdefaultencoding('utf8')
```

Comment:
@waybarrios please try to find a way to accomplish the same thing directly on the object where you are doing the matching.